### DISCUSSION TITLE
有关 v2ray dns 处理的一点想法

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [v2fly/v2ray-core#560] [Ideas]
现在 v2ray 的 dns 协议支持 udp dns 和 DoH，但是没有对 tcp dns 提供支持。而且 v2ray 对待 DoH 相比 udp dns 有额外的选项：目前只有 DoH 可以选择是否通过 Routing/Outbound 等组件

希望未来可以增加 tcp dns 的支持，以及 tcp/udp dns 也可以选择不通过 Routing/Outbound 直接请求

**Comment by kslr** [comment]
welcome pr

**Comment by rurirei** [comment]
#415
